analyzer/analyzer.go

```diff
@@ -37,6 +52,16 @@ type CallStack struct {
 	CallStack *CallStack `json:"callStack,omitempty"` // The trace of calls leading to this source.
 }
 
+func (src *CallStack) Hash() string {
+	sub := src.Function
+	if src.CallStack != nil {
+		sub = fmt.Sprintf("%s:%s", sub, src.CallStack.Hash())
+	}
+	h := sha256.New()
```

Comment:
Why are we doing hashing again in Callstack. Hashing is expensive and should be done just once for the Issue. From callstack, we can just return the function name recursively.